charters/UploadService/charter.md

```diff
@@ -0,0 +1,80 @@
+# Upload Service
+
+## Description
+The Human Cell Atlas (HCA) Data Coordination Platform (DCP) Upload Service provides a file staging and validation facility. Upload Areas are created and deleted using a [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) Application Programming Interface (API), which is secured so only the DCP Ingestion Service may use it. It stages files into cloud storage and computes checksums for the files. The validation service can run arbitrary validation jobs against files.
+
+## Objectives
+1. Provide temporary storage for staging of files by submitters during ingestion into DCP Ingestion Service and subsequent storage in the Data Storage Service (DSS)
+1. Provide tools a command line interface (CLI) and API to upload files to staging areas
+1. Provide a facility to validate files by running an arbitrary workload against the file to be used by Ingest for validation
```

Comment:
what do you mean by `arbitrary workload`?

Reply:
I would rewrite this "Provide a facility to validate files of various types that accepts user generated validators that meet upload service convention"